src/components/AppLayout/NavHeader.tsx

```
@@ -96,6 +96,7 @@ const NavHeader = ({ className, sidedrawer, setSidedrawer, displayName, isVerifi
 	const [openAddressModal, setOpenAddressModal] = useState<boolean>(false);
 	const dispatch = useDispatch();
 	const { resolvedTheme: theme } = useTheme();
+	const [isIdentityModalOpen, setIsIdentityModalOpen] = useState(false);
```

ESLint

[error] 'isIdentityModalOpen' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)